db/migrate/20190718124955_create_activities.rb

```diff
@@ -0,0 +1,11 @@
+class CreateActivities < ActiveRecord::Migration
+  def change
+    create_table :activities do |t|
+      t.references :company
```

Comment:
t.integer = company_id, index: true ...can use this?

Reply:
Done